subworkflows/local/metagenomics_postprocessing.nf

```diff
@@ -17,7 +17,10 @@ workflow METAGENOMICS_POSTPROCESSING {
     if ( params.metagenomics_run_postprocessing && params.metagenomics_profiling_tool == 'malt' ) {
 
         //maltextract doesnt accepts a meta param in the first input channel, so remove it
-        ch_maltextract_input = ch_postprocessing_input.map{it[1]}
+        ch_maltextract_input = ch_postprocessing_input.first().map {
```

Comment:
this change is rather drastic. Are you sure we only need the first item of the channel?

Reply:
This is related to a change made in the metagenomics_profiling.nf file -- which moves all the output rma6 files into a single list for both the parallel and single malt execution and uses .combine to put that on each ch_postprocessing_input entry... but only one of the rma6 file lists is needed. 

therefore the first()

I will go over this again and make sure everything is workign as expected -- still need to get rma2info --> taxpasta working for malt since it is erroring out on my end. thats this friday. :)

Reply:
Ah that makes sense. Thanks for clarifying!

yes, I remember that rma2info&taxpasta didnt work for me as well because rma2info was creating an empty file. I suspected no hits with malt, but didnt test further. Thanks for loking into that as well!